shardingsphere-jdbc/shardingsphere-jdbc-core/src/main/java/org/apache/shardingsphere/driver/jdbc/core/resultset/ShardingSphereResultSet.java

```diff
@@ -372,4 +376,21 @@ public Object getObject(final String columnLabel) throws SQLException {
         int columnIndex = columnLabelAndIndexMap.get(columnLabel);
         return mergeResultSet.getValue(columnIndex, Object.class);
     }
+
+    @Override
+    public <T> T getObject(final int columnIndex, final Class<T> type) throws SQLException {
```

Comment:
Is it possible to support more types?

Reply:
good point. I will add more type support in another  pull request

Reply:
ok, will remove it